couchdb/client.py

```diff
@@ -166,6 +167,18 @@ def version(self):
         status, headers, data = self.resource.get_json()
         return data['version']
 
+    def major(self):
```

Comment:
I'd prefer to do `version_info()`, which would return a tuple of the version components (e.g. `(2, 0, 0)`. This mirrors `sys.version_info` (which is apparently a namedtuple, maybe we can steal that pattern?). This seems a bit more general than just having `major`, for which the name also seems a little arbitrary/out-of-place.